Library/Homebrew/cmd/search.rb

```diff
@@ -127,7 +125,7 @@ def search_tap(user, repo, regex_or_string)
     regex = regex_or_string.is_a?(String) ? /^#{Regexp.escape(regex_or_string)}$/ : regex_or_string
 
     if (HOMEBREW_LIBRARY/"Taps/#{user.downcase}/homebrew-#{repo.downcase}").directory? && \
-       user != "Caskroom"
+       user != "caskroom"
```

Comment:
It's just aesthetic. The GitHub username is lowercase, so it doesn't make sense to capitalize it when on the other hand we are lowercasing `homebrew`, which actually is capitalized.